server/backend/api/app/main.py

```diff
@@ -129,6 +130,13 @@ def main():
     dependencies=[Depends(router_guards.authorize)],
     tags=["FAM User Terms and Conditions"],
 )
+app.include_router(
+    router_user.router,
+    prefix=apiPrefix + "/user",
+    dependencies=[Depends(router_guards.verify_api_key_for_update_user_info)],
```

Comment:
Protect the new api with an api key, because if we want to call this api by setting up a cron job in API Gateway in the future, it's hard to do a user authentication. Pass an api key will be easier.